website/blog/2025/03-18-Gardener-KubeCon-CloudNativeCon-Europe-2025-Announcement.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Gardener at KubeCon + CloudNativeCon Europe, London 2025"
+linkTitle: "Gardener at KubeCon + CloudNativeCon Europe, London 2025"
+newsSubtitle: Mar 18, 2025
+publishdate: 2025-03-18
+authors:
+- name: Sonu Kumar Singh
+  email: sonu.kumar.singh02@sap.com
+  avatar: https://avatars.githubusercontent.com/acumino
+aliases: ["/blog/2025/03/18/01"]
+---
+
+# Gardener at KubeCon + CloudNativeCon Europe, London 2025
+
+The open-source Kubernetes project [Gardener](https://gardener.cloud/) is set to showcase its cutting-edge Kubernetes-as-a-Service (KaaS) capabilities at [KubeCon + CloudNativeCon Europe](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/) 2025 in London.
```

Comment:
```suggestion
The open-source project [Gardener](https://gardener.cloud/) is set to showcase its cutting-edge Kubernetes-as-a-Service (KaaS) capabilities at [KubeCon + CloudNativeCon Europe](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/) 2025 in London.
```

Otherwise it sounds like gardener is part of the kubernetes organization which is not the case.